libs/designer/src/lib/ui/Designer.tsx

```diff
@@ -22,15 +22,15 @@ import { setLayerHostSelector } from '@fluentui/react';
 import type { WorkflowNodeType } from '@microsoft/utils-logic-apps';
 import { useWindowDimensions, WORKFLOW_NODE_TYPES, useThrottledEffect } from '@microsoft/utils-logic-apps';
 import { useCallback, useEffect, useMemo, useState } from 'react';
-import KeyboardBackend, { isKeyboardDragTrigger } from 'react-dnd-accessible-backend';
+import KeyboardBackendFactory, { isKeyboardDragTrigger } from 'react-dnd-accessible-backend';
 import { HTML5Backend } from 'react-dnd-html5-backend';
 import { DndProvider, createTransition, MouseTransition } from 'react-dnd-multi-backend';
 import { useDispatch, useSelector } from 'react-redux';
-import { ReactFlow, ReactFlowProvider, useNodes, useReactFlow, useStore, BezierEdge } from 'reactflow';
-import type { NodeChange } from 'reactflow';
+import { Background, ReactFlow, ReactFlowProvider, useNodes, useReactFlow, useStore, BezierEdge } from 'reactflow';
+import type { BackgroundProps, NodeChange } from 'reactflow';
 
 export interface DesignerProps {
-  graphId?: string;
```

Comment:
Note: This set of props was not used anywhere, so should not be any issue removing `graphId`.